pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun.go

```diff
@@ -85,141 +81,79 @@ func (tr *TrustedTaskRun) Validate(ctx context.Context) (errs *apis.FieldError)
 func (tr *TrustedTaskRun) SetDefaults(ctx context.Context) {
 }
 
-func (tr *TrustedTaskRun) VerifyTask(
+func (tr *TrustedTaskRun) verifyTaskRun(
 	ctx context.Context,
 	k8sclient kubernetes.Interface,
 	tektonClient versioned.Interface,
 ) (errs *apis.FieldError) {
 	logger := logging.FromContext(ctx)
-	logger.Info("Verifying Resources")
+	logger.Info("Verifying TaskRun")
 
-	if tr.ObjectMeta.Annotations == nil {
-		return apis.ErrMissingField("annotations")
+	if tr.Spec.TaskRef == nil {
```

Comment:
nit: add a comment explaining why we're ignoring this.

Reply:
Oh thanks! I realized that I added the comment but add it in a wrong place (line 96), will move it here.